### ISSUE TITLE
loss 관련 질문입니다.

### ISSUE THREAD

**Issue by HyunjungPark** [seominseok0429/Real-world-Anomaly-Detection-in-Surveillance-Videos-pytorch#12]
안녕하세요!
코드 이해하기 쉽게 짜주셔서 도움을 많이 받고 있습니다. 감사합니다~!
loss 코드 중에 anomaly_index/normal_index를 randperm으로 한 부분은 잘 이해가 되지 않는데, 혹시 이렇게 해야하는 이유가 있나요..?

**Comment by seominseok0429** [comment]
고정적인 숫자로 가져오면 선택이 안되는 클립이 존재 할 수 있기 때문입니다. 늘 랜덤하게 랜덤한 갯수 만큼의 클립을 가져오는 것은 데이터 어그멘테이션 이라고 생각하시면 될것 같습니다

**Comment by Chuttyboy** [comment]
> You can think of it as data augmentation.
> 
> For example, it is better to get 30 randomly than to bring 1 to 30.

@HyunjungPark  @seominseok0429  

After training how to test video with the trained model ?  can you please help me 

Thanks

**Comment by seominseok0429** [comment]
@Chuttyboy  Added visualization code. Check out the link below.

https://github.com/seominseok0429/Real-world-Anomaly-Detection-in-Surveillance-Videos-pytorch/blob/main/vis.py

Thanks

**Comment by Chuttyboy** [comment]
> @Chuttyboy Added visualization code. Check out the link below.
> 
> https://github.com/seominseok0429/Real-world-Anomaly-Detection-in-Surveillance-Videos-pytorch/blob/main/vis.py
> 
> Thanks

@seominseok0429  @HyunjungPark  

Thanks for adding visualization code. How to save models after training 

```
checkpoint = torch.load('./weight/RGB_Kinetics_16f.pth')
model.load_state_dict(checkpoint['state_dict'])
checkpoint = torch.load('./weight/ckpt.pth')
classifier.load_state_dict(checkpoint['net'])
```

can you please explain what happened here and how to download `RGB_Kinetics_16f.pth` 

Thanks

**Comment by seominseok0429** [comment]
This is the modified download link. https://drive.google.com/file/d/1xYsBiCSmXjE0BwoiH_Bcm4AWaA1pESHF/view?usp=sharing
